src/eventConsumer/index.ts

```diff
@@ -18,4 +18,5 @@ export const eventConsumer: {
   [EventType.ACCOUNT_DELETION]: userEventConsumer,
   [EventType.ACCOUNT_EMAIL_UPDATED]: userEventConsumer,
   [EventType.PROSPECT_DISMISS]: prospectEventConsumer,
+  //collections-event: collections
```

Comment:
what is this?

Reply:
ah it came up with discussion with herraj, can remove this